tests/unit/test_processing.py

```diff
@@ -388,6 +437,14 @@ def test_processor_with_all_parameters(sagemaker_session):
     sagemaker_session.process.assert_called_with(**expected_args)
 
 
+def test_processing_job_from_processing_arn(sagemaker_session):
+    processing_job = ProcessingJob.from_processing_arn(
+        sagemaker_session=sagemaker_session,
+        processing_job_arn="arn:aws:sagemaker:dummy-region:dummy-account-number:processing-job/dummy-job-name",
+    )
+    assert isinstance(processing_job, ProcessingJob)
```

Comment:
should this also check that attributes are set correctly?

Reply:
Added more checks for the attributes.